pyanaconda/modules/storage/devicetree/model.py

```diff
@@ -541,7 +541,8 @@ def make_mtab(self, chroot=None):
         if os.path.exists(path):
             os.unlink(path)
 
-        os.symlink(target, path)
+        if not os.path.exists(path):
+            os.symlink(target, path)
 
```

Comment:
```suggestion

        try:
        if not os.path.exists(path):
            os.symlink(target, path)
        except FileExistsError:
            # We just need path to exist.  It doesn't matter if it's been created by something
            # else after we unlinked it above.
            pass

```
Better to use try except to avoid race conditions.